torch/_torch_docs.py

```diff
@@ -3571,6 +3571,34 @@ def merge_dicts(*dicts):
     tensor([15, 30, 15])
 """.format(**common_args))
 
+add_docstr(torch.ldexp, r"""
+ldexp(input, other, *, out=None) -> Tensor
+
+Multiplies :attr:`input` by 2**:attr:`other`.
+
+.. math::
+    \text{{out}}_i = \text{{input}}_i * 2^\text{{other}}_i
+""" + r"""
+
+Typically :attr:`input` contains mantissas to be multiplied by the specified specified
+integral powers of two in :attr:`other`, in order to construct floating point numbers.
```

Comment:
Grammar nit: you may want to reverse this sentence:

"Typically this function is used to construct floating point numbers by multiplying mantissas in :attr:\`input\` with integral powers of two in :attr:\'other\'."

Reply:
> Grammar nit: you may want to reverse this sentence:
> 
> "Typically this function is used to construct floating point numbers by multiplying mantissas in :attr:`input` with integral exponents in :attr:'other'."